HHS/Head-Start-TTADP
# [Prod] Update CircleCI image, handle select overflow, resize filter, grant numbers on objective table

## Description of change

- Fix for "empty" filter state on the various filter menus
- Filter select handles overflow more gracefully
- Remove the default date from the goals & objectives page
- Also, on the goals & objectives tab, added a new column "grant numbers" and adjusted the styling to match the new mockup
- Update the virtual machine's OS where our app lives

## How to test

- Remove all filters from the filter menu, the filter menu will not collapse and look weird.
- Add a few options to the filter multiselect, note how it displays (and you can still operate the menu as needed)
- Visit the goals & objectives page and note that there is no default date, and that the objectives table has a grant numbers column


## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-700
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-789
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-785
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-821
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-801

## Checklists

### Every PR


- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
